### PR TITLE
fix: facephoto as hardlimit when camera not found

### PR DESCRIPTION
**Pull request by RafaelTaranto**

### CODE REVIEW

**Comment by RafaelTaranto** [comment]
[Do not bail out if customer camera KYC option enabled and machine doesn't have customer camera](https://trello.com/c/WVXE4dCQ/47-do-not-bail-out-if-customer-camera-kyc-option-enabled-and-machine-doesnt-have-customer-camera)

File: lib/scanner.js [hunk 1]
```diff
@@ -50,6 +51,19 @@ function isOpened () {
   return opened
 }
 
+function hasCamera (mode) {
+  const cameraConf = mode === 'facephoto' ? 'frontFacingCamera' : 'scanner'
+
+  const device = _.get(`${cameraConf}.device`, configuration)
+  const modeConfig = _.get([cameraConf, mode], configuration)
```
Comment (joshmh) [inline lib/scanner.js]: Shouldn't we check for null device?

**Comment by naconner** [comment]
[Account for Santo Tirso on upboard/gaia device_config.json](https://trello.com/c/segYPcYc/3-account-for-santo-tirso-on-upboard-gaia-deviceconfigjson)

File: lib/scanner.js (hunk 1 above)
Comment (RafaelTaranto) [reply lib/scanner.js]: Good point, the try catch would handle this case but I added a new test for it.

**Pull request merged**